### ISSUE TITLE
Sort Accounts with customised order in balance sheet

### ISSUE THREAD

**Issue by ileodo** [beancount/fava#1657]
Is there a way to sort accounts by using some customised order in balance sheet?

**Comment by yagebu** [comment]
No, there is not, they are sorted alphabetically. What kind of order where you thinking of?

**Comment by ileodo** [comment]
a customised order, would be good that use some meta data field in the `open` directive to indicate the order of the accounts, then respect them in fava

**Comment by MJochim** [comment]
Maybe starting all account names with a number is an option for you?

**Comment by floriskruisselbrink** [comment]
> Maybe starting all account names with a number is an option for you?

Account name parts (seperated by ':') cannot start with a number, so this is not really an option.

I too would like to be able to specify some kind of metadata as sortkey.

**Comment by MJochim** [comment]
They can, and I use that a lot (sometimes for sorting, but mostly for other reasons). They just can't start with a lower-case letter.

I'll admit it's a little hacky, but it works.
